widgetssdk/src/main/java/com/glia/widgets/StringKeyPair.kt

```diff
@@ -1,23 +1,34 @@
 package com.glia.widgets
 
-/**
- * Retrieve custom locales strings that should have placeholders replaced with real values.
- */
-data class StringKeyPair(val key: StringKey, val value: String)
+import android.os.Parcelable
+import kotlinx.parcelize.Parcelize
 
-/**
- * List placeholders for [StringKeyPair].
- */
+@Deprecated("Feature removed for internal use only")
```

Comment:
This wording is ambiguous to me. "Feature is for internal use only" sounds more clear in my opinion.

Reply:
Or lets at least transform into two sentences: "Feature was removed. It's for internal use only now."